redisinsight/ui/src/components/navigation-menu/components/notifications-center/styles.module.scss

```diff
@@ -0,0 +1,160 @@
+@import '@elastic/eui/src/global_styling/mixins/helpers';
+@import '@elastic/eui/src/global_styling/index';
+
+.wrapper {
+  position: relative;
+  display: flex;
+  @media screen and (max-width: 767px) {
```

Comment:
instead of hardcode number of resolution you can use `@include euiBreakpoint("xs", "s")`

Reply:
done